services/common/boltzmann/response.js

```diff
@@ -75,18 +77,32 @@ function error(err, status = 500, extraHeaders = {}) {
   if (typeof err === 'string') {
     err = { message: err, code: 'ENOTSUPPLIED' };
   }
+
+  if (isDev()) {
+    err.trace = new Error().stack;
+  }
```

Comment:
If we're in dev mode, add a `trace` property to all outgoing errors. This makes it easier to track down errors when they happen: the registry client will log traces coming from `storage`.